### ISSUE TITLE
CPU usage stays high even after closing the video file

### ISSUE THREAD

**Issue by pickpocket99** [lhc70000/iina#1235]
So I got myself a new MacBook 12" and wanted to test out the new HEVC capabilities. To do so I downloaded a couple of 4k video files. One of which is http://jell.yfish.us/media/jellyfish-400-mbps-4k-uhd-hevc-10bit.mkv 

In the beginning everything seemed fine and I was impressed how my little machine could handle the video with ease. Sadly something changed and the CPU get's put into turbo-boost immediately. What's even worse, after closing the window it stays in boost-mode until I quit IINA completely. 

I'm not sure what could have led to this behavior since I didn't install any new software or made any changes. 

Same thing happens with this file: http://downloads.4ksamples.com/downloads/sample-Elysium.2013.2160p.mkv
Which played just fine only a few days ago.

Just to make sure I did some more testing with those HEVC 10 Bit jellyfish files.

60mbps play fine and doesn't boost. 

UPDATE: The 60mbps files has started to exhibit the same behavior.

90, 200, 250 and 300 exhibit the same issues.

Similar issue here: http://downloads.4ksamples.com/downloads/SES.Astra.UHD.Test.2.2160p.UHDTV.HEVC.x265-LiebeIst.mkv
Only difference being, a couple of seconds after closing the window core frequency goes back to normal.

**System and IINA version:**

- macOS 10.13.1
- IINA 0.0.14.1

**Expected behavior:**

**Actual behavior:**

- Crash report:
```

```

- MPV log:
```

```

**Steps to reproduce:**
1. Open one of the aforementioned files
2. Look at CPU usage in activity monitor and frequency in Intel power gadget
3. Close window, CPU usage stays the same

**Comment by pickpocket99** [comment]
Ok, so I was bored and felt like procrastinating and did some testing, where in conclusion I can say I have no idea what's going on.

Here's some screenshots of how the CPU was taxed.

First I used Sony's [camp](http://4kmedia.org/sony-camping-in-nature-4k-demo/) demo 
1. quicktime
![sony camp demo in quicktime](https://user-images.githubusercontent.com/33791182/33187309-41e598d6-d091-11e7-8276-29cee6324b9c.png)

2. iina
![sony camp demo in iina](https://user-images.githubusercontent.com/33791182/33187318-4cbc60a0-d091-11e7-815a-4951bf3f9047.png)

And later on it was actually working fine-ish in iina.

So next I picked Sony's [whale](http://4kmedia.org/sony-whale-in-tonga-hdr-uhd-4k-demo/) demo clip, and this one was really weird.

1. quicktime
![sony whale quicktime](https://user-images.githubusercontent.com/33791182/33187389-b38c5b64-d091-11e7-97f4-26d442176abd.png)

2. iina
![sony whale iina](https://user-images.githubusercontent.com/33791182/33187393-bc7b2d0e-d091-11e7-8148-f8391d7e031b.png)

so everything's fine with iina? not so fast.

3. I didn't even quit the program, just let the cpu cool down. And then this happened. Dropped about 1800 frames, video came to a halt, sound cut out, and there where still 40 seconds left when I stopped it.
![sony whale iina 1800 fps drops,  40 seconds left](https://user-images.githubusercontent.com/33791182/33187400-cf42e8a0-d091-11e7-8617-6a7c9707aa83.png)

4. guess what, the same instance of iina can play the file just fine.

![sony whale iina](https://user-images.githubusercontent.com/33791182/33187511-76c09546-d092-11e7-990a-095cb49afb3d.png)

And just to finish things off I did some testing with this file: http://4kmedia.org/philips-uhd-4k-beach-life-demo/ which had been making some issues as well.

1. quicktime. everything is fine

![philips 1](https://user-images.githubusercontent.com/33791182/33187561-ee67b340-d092-11e7-94c6-5ecdc4707a02.png)

2. iina. not so great 

![philips 2](https://user-images.githubusercontent.com/33791182/33187570-fc47cc3e-d092-11e7-9aa3-eb3d73711208.png)

3. iina. pretty good

![philips 3](https://user-images.githubusercontent.com/33791182/33187584-16b8721c-d093-11e7-8769-a4706b4e6cdb.png)

4. iina just to make sure, if this wasn't just a fluke

![philips 4](https://user-images.githubusercontent.com/33791182/33187592-24198040-d093-11e7-98c8-ae5a184fa461.png)

looks like it wasn't.

Additionally I played the Elysium file, from my initial post, and one time the CPU went into overdrive, and another time everything went fine.

Also the 60mbit jellyfish file has become sort of a lottery, sometimes there's almost no issues, and the next time the cpu works like crazy.

**Comment by lhc70000** [comment]
Moved to #1724.

tl;dr This may be fixed in beta 2.
